x-pack/test/functional/apps/lens/group5/drag_and_drop.ts

```diff
@@ -251,7 +251,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await lens.assertFocusedField('clientip');
       });
       it('should duplicate an element in a group', async () => {
-        await lens.dimensionKeyboardDragDrop('lnsXY_yDimensionPanel', 0, 1);
+        await lens.dimensionKeyboardDragDrop('lnsXY_yDimensionPanel', 0, 2);
```

Comment:
Ah this is the issue @dej611. I'm not sure how this change got added in your original PR but it was done so when these tests were skipped so this change was never validated.

```suggestion
        await lens.dimensionKeyboardDragDrop('lnsXY_yDimensionPanel', 0, 1);
```

The steps here should be `1` which will place the group on the duplicate/add drop site, but with `2` it places it back in its original order which results in a noop.

This test failure then caused the proceeding tests to fail.